workflows/methylation/methylation-preprocess.wdl

```diff
@@ -0,0 +1,98 @@
+version 1.1
+
+workflow methylation_preprocess {
+    meta {
+        description: "Process raw IDAT files from the Illumina methylation array for a single sample"
+        outputs: {
+            beta_swan_norm_unfiltered: "Normalized beta values for all probes",
+            beta_swan_norm_unfiltered_genomic: "Normalized beta values for all probes that map to the genome",
+            annotation: "Annotation table for probes.  Contains genomic coordinates and sequence and other information about the probes.",
+            beta_unnorm: "Non-normalized beta values",
+            cn_values: "Copy number values",
+            m_values: "M values",
+            probe_names: "Probe names found on the array",
+            sample_names: "Sample name found on the array",
```

Comment:
Is this informative? I don't get why I'd want it output. Especially because this will just be a single sample, right?

Reply:
In the single-sample context, it probably isn't informative. Especially, since it should match the prefix portion of the file name.

Reply:
So should we remove it?